src/platforms/posix/drivers/df_mpu9250_wrapper/df_mpu9250_wrapper.cpp

```diff
@@ -173,9 +194,11 @@ DfMpu9250Wrapper::DfMpu9250Wrapper(/*enum Rotation rotation*/) :
 	_fifo_corruption_counter(perf_alloc(PC_COUNT, "mpu9250_fifo_corruptions")),
 	_gyro_range_hit_counter(perf_alloc(PC_COUNT, "mpu9250_gyro_range_hits")),
 	_accel_range_hit_counter(perf_alloc(PC_COUNT, "mpu9250_accel_range_hits")),
+	_mag_range_hit_counter(perf_alloc(PC_COUNT, "mpu9250_mag_range_hits")),
```

Comment:
I am going to change this counter to a FIFO overflow counter for the mag.  I will make sure it get's updated.